site/styles/main.scss

```diff
@@ -357,3 +357,10 @@ s72-classification-label {
 h1, h2, h3, h4, h5, h6, p, .title {
   word-wrap: break-word;
 }
+
+.noscroll {
+  overflow: hidden;
+  @include media-breakpoint-up(md) {
```

Comment:
Why the different behaviour?

Reply:
So if the user resizes the window back to desktop, they can scroll.